Internal: Release/3.8.0

matipojo opened the pull request.

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

mark-elementor committed:
Internal: ESLint - Fixes (#19094)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

Commit:
Tweak: Custom links in menu items should be relative to the site address in Kit import [ED-6947] (#18668)

* Tweak: Menu items with custom links should imported relative to the site address when importing a Kit [ED-6937]

* CR fixex

* Fixes

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

Commit:
Internal:  Testing Finder using PHPUnit [ED-6095] (#17334)

* Internal:  Testing Finder using PHPUnit [ED-6095]

* Changes

Co-authored-by: yossi-elementor <87371461+yossi-elementor@users.noreply.github.com>

yossi-elementor committed:
Tweak: UTM missing on the import-export error message [ED-6317] (#19127)

Replace error link of import export manifest error

Co-authored-by: Guy Cohen <guyc@elementor.com>

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

mark-elementor committed:
Tweak: Notes was added to the right click context menu [ED-7411] (#19026)

* Tweak: Notes was added to the right click context menu [ED-7411]

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

mark-elementor committed:
Internal: Notes menu item in Settings moved to the Notes module [ED-7649] (#19138)

mark-elementor committed:
Fix: Kit Library, default direction for Featured sorting to ASC, hide button [ED-7281] (#18760)

Commit:
Fix: Insert library button is clickable while inserting a template [ED-7008] (#18741)

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/elementor/elementor?pullRequest=19184) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/elementor/elementor?pullRequest=19184) before we can accept your contribution.<br/>**3** out of **5** committers have signed the CLA.<br/><br/>:white_check_mark: matipojo<br/>:white_check_mark: ManorHazaz<br/>:white_check_mark: mark-elementor<br/>:x: elementorbot<br/>:x: yossi-elementor<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/elementor/elementor?pullRequest=19184) it.</sub>

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

Commit:
Tweak: Increased Notes discoverability in the editor [ED-7726] (#19191)

davseve committed:
Fix: Go pro link is too wide in export kit tool [ED-7575] (#19200)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

udidol committed:
Tweak: Convert "Template Kits" to "Website Kits" phrasing throughout the plugin [ED-7673] (#19219)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

hein-obox committed:
Fix: Divider widget is not working as expected [ED-7722] (#19224)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

hein-obox committed:
Fix: Align-self set to stretch is not working as expected in Container (#17052) [ED-7359] (#19222)

matipojo committed:
Merge remote-tracking branch 'elementor/release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

mark-elementor committed:
Fix: Notes - Promotion dialog not opening in Panel menu [ED-7744] (#19192)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

Commit:
Internal: Fix linter warning in experiments manager [ED-7877] (#19339)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

mark-elementor committed:
Fix: Align notices to RTL sites [ED-4809] (#19337)

elementorbot committed:
Merge branch 'release/3.7.0' into release/3.8.0

Commit:
Fix: Export kit doesn't work in a Multisite Network [ED-7696] (#19146)

davseve committed:
Tweak: Add to the GoPro link URL from the Kit Library more UTM parameters [ED-7745] (#19336)

davseve committed:
Internal: Handle merging conflict between develop and 3.8 [ED-7961] (#19466)

davseve committed:
Internal: Handle merging conflict between develop and 3.8.0 [ED-7691] (#19478)

Commit:
Tweak: Import/Export CLI and UI mechanisms were merged into a unified service [ED-7157] (#19044)

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

matipojo committed:
Internal: Merge develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

Commit:
Fix: Importing fails when post type file is missing [ED-8002] (#19518)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

TzviRabinovitch committed:
Internal: Experiment - Introduce internal experiment [ED-8016] (#19548)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

arielk committed:
Tweak: Increase inputs in Replace URL tool to support long URLs [ED-8030] (#19565)

StyleShit committed:
Internal: Experiments Manager - Prevent depending on a hidden experiment [ED-8028] (#19561)

arielk committed:
Fix: The title of editor control are not responsive [ED-8034] (#19577)

Co-authored-by: Mati Horowitz <21468434+matipojo@users.noreply.github.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

arielk committed:
Tweak: Add Dynamic Tags for Global Colors [ED-8050] (#19598)

Co-authored-by: Mati Horowitz <21468434+matipojo@users.noreply.github.com>

Commit:
Fix: Import/Export - WooCommerce products are not being imported [ED-7932] (#19603)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

StyleShit committed:
Tweak: Improve the UX of dependencies between experiments [ED-7494] (#19037)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

Commit:
Fix: Import/Export CLI - Import not overriding all templates conditions [ED-8045] (#19607)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

yossi-elementor committed:
Fix: Changed link to https (#19630)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

rodolphebertozzo committed:
Tweak: Added clarification for the 'a' tag to the container element (#19599)

Added a note to clarify for users who are not familiar with HTML that the a tag allows you to add a link before clicking on it.
![Capture d’écran 2022-08-27 202527](https://user-images.githubusercontent.com/56270626/187043380-12f10f1f-23ed-496f-acb1-749e386bc1b0.png)

Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>

elementorbot committed:
Bump 3.8.0-cloud1

udidol committed:
Internal: Remove `__wakeup` PHPUnit test which causes an exception in the CI's PHP 7.4 [ED-8101] (#19645)

rodolphebertozzo committed:
Fix: Translation (#19652)

The native translation method of Wordpress esc_html__() is missing for adding the HTML tag 'a' to the container

arielk committed:
Tweak: Add more units to icon and icon-box widgets [ED-8103] (#19651)

Co-authored-by: Mati Horowitz <21468434+matipojo@users.noreply.github.com>

rodolphebertozzo committed:
Tweak: Added em unit for border radius to layout elements (#19491)

Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>

TzviRabinovitch committed:
Internal: Merge loop builder CORE to 3.8 CORE [ED-7714] (#19193)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

FlyingDR committed:
Tweak: Synchronous elements initialization may produce long JavaScript tasks (#15229)

Fixed #15228

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-cloud2

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

TzviRabinovitch committed:
Internal: Added `hidden` constant to core experiments manager [ED-8129] (#19678)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

StyleShit committed:
Tweak: Improved performance of Inline Fonts Icons experiment [ED-8118] (#19672)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

TzviRabinovitch committed:
Fix: lint issue

hein-obox committed:
Tweak: Changed Full-width and Boxed content width functionality in Container [ed-7867] (#19385)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

rodolphebertozzo committed:
Tweak: Responsive text-align in Columns & Sections [ED-6803] (#16215)

I tested and the predefined parameters are saved when you switch to another breakpoint
![align](https://user-images.githubusercontent.com/56270626/137280398-c99f14e3-a6d0-402a-876f-85178fe67817.PNG)
![align2](https://user-images.githubusercontent.com/56270626/137280400-eac6bf7e-a5ab-48d5-8fe7-d75dc2734413.PNG)
![align3](https://user-images.githubusercontent.com/56270626/137280401-5ccf357a-dd82-4683-8848-c1482a417cea.PNG)
fix: https://github.com/elementor/elementor/issues/13199

udidol committed:
Internal: Extract `/core/apps` into the root directory [ED-7996] (#19576)

rodolphebertozzo committed:
Tweak: Added text stroke for some widgets [ED-6780] (#18533)

- Tweak: Added text stroke to number for counter widget
- Tweak: Added text stroke to title for counter widget
- Tweak: Added text stroke to text for divider widget
- Tweak: Added text stroke to title for image box widget

Co-authored-by: Ariel Klikstein <ariel@elementor.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

rodolphebertozzo committed:
Tweak: Add more units to testimonial and image-box widgets (#19694)

Tweak: Add more units to testimonial and image-box widgets

rami-elementor committed:
Tweak: Add labels to font weight numeric values [ED-7670] (#18990)

## PR Checklist
- [x] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] Other... Please describe: i18n

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add labels to font weight numeric values

## Description
An explanation of what is done in this PR

* Tweak: Add labels to font weight numeric values

Fixes #18761

Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

udidol committed:
Fix: Kit Library Connect doesn't work in 3.8.0 with production Pro [ED-8156] (#19712)

maork-elementor committed:
Tweak: Change default Google fonts loading method to "Swap" [ED-7896] (#19692)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

Commit:
Tweak: Allow removing the last imported Kit [ED-6987] (#19510)

Commit:
Fix: Import resolver page is being displayed on empty import conflict [ED-8106] (#19661)

KingYes committed:
Tweak: Soft deprecation for the wrong using with widgets API [ED-8171] (#19736)

![image](https://user-images.githubusercontent.com/902548/188905601-8f778e80-975e-4587-911b-b2fe5f7a1b71.png)

mark-elementor committed:
Fix: Posts widget - query is not being imported correctly when importing a Kit [ED-4687] (#19636)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

rami-elementor committed:
Tweak: Add size units to "Typography" > "Letter Spacing" [ED-8184] (#19745)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Add size units to "Typography" > "Letter Spacing"

## Description
An explanation of what is done in this PR

* Tweak: Add size units to "Typography" > "Letter Spacing"

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #19726

StyleShit committed:
Internal: Remove unused code in Inline Fonts Icons experiment [ED-8139] (#19693)

matipojo committed:
Revert "Tweak: Soft deprecation for the wrong using with widgets API [ED-8171]" (#19755)

Reverts elementor/elementor#19736, it changes a public method signature.

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

Commit:
Fix: Improter WordPress root write permissions check causes import failures [ED-5913] (#19380)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

Commit:
Fix: Import Export admin page - revert button causing JS console error [ED-8196] (#19767)

Commit:
Fix: CSS minified files not generated on build [ED-8199] (#19768)

Commit:
Internal: Collect Kit Revert data into the usage data [ED-7426] (#19741)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

yossi-elementor committed:
Fix: Editor menu items not working [ED-8168] (#19733)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

KingYes committed:
Tweak: Added migrate script to handle PHP8 type error on custom image size [ED-8166] (#19725)

Continue from #19722

arielk committed:
Internal: Change CTA to Upgrade instead of See it in action (#19785)

Co-authored-by: Mati Horowitz <21468434+matipojo@users.noreply.github.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

hein-obox committed:
Tweak: Container - Move editor code to an editor scss file [ED-7940] (#19421)

louiswol94 committed:
Tweak: Change the Container experiment status to Beta [ED-8085] (#19788)

maork-elementor committed:
Fix: Experiments - Status change callback not working [ED-8201] (#19781)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

maork-elementor committed:
Internal: Tests - Status change callback test not working [ED-8201] (#19801)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-cloud3

hein-obox committed:
Tweak: Changed right-click functionality when adding a new Container [ED-7870] (#19684)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-cloud4

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

mark-elementor committed:
Fix: On import - replace dynamic content fix [ED-8280] (#19847)

TzviRabinovitch committed:
Tweak:  Adjust the tag display in the experiments screens [ED-8277] (#19846)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-beta1

elementorbot committed:
Bump 3.8.0-cloud5

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

StyleShit committed:
Fix: Connect - Connect notice appearing in a wrong place [ED-8249] (#19849)

WordPress moves notices blindly based on tag name:

https://github.com/WordPress/WordPress/blob//5.9-branch/wp-admin/js/common.js#L1074-L1083

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

StyleShit committed:
Internal: Remove unnecessary quote in Inline Fonts experiment [ED-8309] (#19864)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

StyleShit committed:
Fix: Error on connect with `WP_DEBUG` = true [ED-8252] (#19862)

The call order was wrong and caused the returned data to be invalid.

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-beta2

elementorbot committed:
Bump 3.8.0-cloud6

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

hein-obox committed:
Fix: Container - Apply inner container class to widgets inside the container [ED-8137] (#19686)

Commit:
Fix: Import/Export - Not importing the relations between terms [ED-8265] (#19875)

Co-authored-by: yossi-elementor <87371461+yossi-elementor@users.noreply.github.com>

Commit:
Fix: Import/Export - Can't import on a new site [ED-8240] (#19850)

Co-authored-by: yossi-elementor <87371461+yossi-elementor@users.noreply.github.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

udidol committed:
Fix: Core - Loop template does not change live when editing the design in-place [ED-8371] (#19951)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Core - Loop template does not change live when editing the design
in an in-place editing
* Added filters to allow intervening with document CSS enqueuing and
printing.

rami-elementor committed:
Internal: Add `@deprecated` to deprecated functions (#19961)

## PR Checklist
<!-- 
Please check if your PR fulfills the following requirements:
**Filling out the template is required.** Any pull request that does not
include enough information to be reviewed in a timely manner may be
closed at the maintainers' discretion.
 -->
- [x] The commit message follows our guidelines:
https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md


## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* This PR added `@deprecated` phpDocs to deprecated functions. Will help
IDEs to highlight deprecated functions.

## Description
An explanation of what is done in this PR

* Internal: Add `@deprecated` to deprecated functions

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [x] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [x] Docs have been added / updated (for bug fixes / features)

Fixes #

udidol committed:
Tweak: Changed container class names to shorter names (markup changes!) [ED-8347] (#19962)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Changed container class names to shorter names:
-- Replace `e-container` to `e-con`
-- Replace `e-container--width-boxed` to `e-con-boxed`
-- Replace `e-container--width-full` to `e-con-full-width`
-- Replace `e-container__inner` to `e-con-inner`

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

davseve committed:
Fix: See-it-live button missing on import kit summery [ED-8258] (#19937)

hein-obox committed:
Fix: Container - apply new inner container class [ED-8392] (#19970)

TzviRabinovitch committed:
Revert "Tweak: Changed container class names to shorter names (markup changes!) [ED-8347]" (#19972)

Reverts elementor/elementor#19962

TzviRabinovitch committed:
Revert "Fix: Container - apply new inner container class [ED-8392]" (#19971)

Reverts elementor/elementor#19970

elementorbot committed:
Bump 3.8.0-cloud7

elementorbot committed:
Bump 3.8.0-beta3

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

arielk committed:
Tweak: Add `rem` to gap between elements control [ED-8343] (#20008)

arielk committed:
Fix: Change all browser behavior by dark mode [ED-8212] (#20013)

Before:
<img width="296" alt="image"
src="https://user-images.githubusercontent.com/1778512/194758855-edbee2ad-9204-4845-a63e-c555278d9f49.png">

After:
<img width="296" alt="image"
src="https://user-images.githubusercontent.com/1778512/194758870-831be486-d72e-41bd-a5b0-c2b745f04150.png">

Co-authored-by: Yakir Sitbon <kingyes1@gmail.com>

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-cloud8

elementorbot committed:
Bump 3.8.0-beta4

udidol committed:
Internal: Infra for Pro fix - Loop template doesn't change live when in-place editing [ED-8416] (#20024)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Internal: Infra updates to support Pro fix - Loop template does not
change live when editing the design in-place

TzviRabinovitch committed:
Revert "Fix: First container handle is not centered on an RTL page  [ED-7788]"

This reverts commit b12795e084a2c985cda71f64164dd147c301ee93.

TzviRabinovitch committed:
Fix: First container handle is not centered on an RTL page  [ED-7788]

louiswol94 committed:
Fix: Container isolation: isolate issues (#19834, #19845, #19705) [ED-8353] (#20014)

Commit:
Fix: Kit Library order on WP dashboard menu was changed [ED-8221] (#19969)

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

hein-obox committed:
Fix: Swiper functionality is not working as expected [...] (#19891) [ED-7916] (#19919)

udidol committed:
Fix: Custom background position not inherited on mobile and front end [ED-8329] (#19883)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Custom background position not inherited on mobile and front end
(Fixes: #19924)
* Fix: Responsive controls with responsive conditions whose values are
arrays (Base Multiple) did not apply in the front end for non-desktop
devices.
* Fix: In certain edge cases, third party addons crash pages since
v3.7.7 (Fixes: #19917, #19922 as well as problems raised in the comments
of #19894)

## Description
An explanation of what is done in this PR

* If a responsive control was conditioned upon another responsive
control, and that conditioning control was a Base Multiple control
(which means - the control value was an array(PHP)/object(JS) - such as
Slider controls [size, unit]) - the responsive values were not applied
in the front end

louiswol94 committed:
Fix: Page should scroll to the loop area and not to the top of the page [ED-8447] (#20075)

udidol committed:
Tweak: Changed container class names to shorter names (markup changes!) [ED-8392] (#20073)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [X] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Tweak: Changed container class names to shorter names (markup
changes!) [ED-8347]
* Fix: Apply shorter container class everywhere

[ED-8347]:
https://elementor.atlassian.net/browse/ED-8347?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-beta5

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

matipojo committed:
Merge remote-tracking branch 'elementor/develop' into release/3.8.0

udidol committed:
Fix: Container Full Width values are disappearing after latest version update [ED-8468] (#20111)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Container Full Width values are disappearing after latest version
update
* Tweak: Container - Changed "full width" class name from
`e-container--width-full` to `e-con-full`

KingYes committed:
Merge branch 'master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

udidol committed:
Fix: Responsive controls with responsive conditions don't always apply in the front end [ED-8471] (#20112)

## PR Type
What kind of change does this PR introduce?
<!-- Please check the one that applies to this PR using "x" with no
spaces eg: [x]. -->
- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

* Fix: Responsive controls with responsive conditions did not apply in
the front end for non-desktop devices

KingYes committed:
Merge branch 'master' into develop

KingYes committed:
Merge branch 'develop' into release/3.8.0

KingYes committed:
Merge branch 'master' into develop

KingYes committed:
Merge branch 'develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

louiswol94 committed:
Fix: Transitions functionality is not working as expected in Container (#19913) [ED-8387] (#20127)

KingYes committed:
Merge branch 'develop' of github.com:elementor/elementor into develop

KingYes committed:
Merge branch 'master' into develop

KingYes committed:
Merge branch 'develop' into release/3.8.0

louiswol94 committed:
Fix: Progress Bar layout is breaking when the direction is set to row in a container [ED-8167] (#20126)

KingYes committed:
Fix: Broken site layout on plugin upgrade [ED-8202] (#19974)

KingYes committed:
Revert "Fix: Broken site layout on plugin upgrade [ED-8202]" (#20136)

Reverts elementor/elementor#19974

KingYes committed:
Fix: Broken site layout on plugin upgrade [ED-8202] (#20137)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Bump 3.8.0-cloud9

elementorbot committed:
Bump 3.8.0-beta6

Nevoss committed:
Fix: Clear cache on plugin update only once (#20154)

Nevoss committed:
Internal: Improvement performance of Playwright CI [ED-8530] (#20161)

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

elementorbot committed:
Merge remote-tracking branch 'origin/master' into develop

elementorbot committed:
Merge remote-tracking branch 'origin/develop' into release/3.8.0

TzviRabinovitch committed:
Internal: Changelog 3.8.0 (#20185)

TzviRabinovitch committed:
Update publish-release.yml
